plugins/foreman_maintain/0.2/index.md

```diff
@@ -0,0 +1,405 @@
+---
+layout: plugin
+title: Foreman Maintain 0.2 Manual
+version: 0.2 
+---
+
+# Foreman-maintain
+
+For general information about foreman-maintain, please check the [github repo](https://github.com/theforeman/foreman_maintain)
+
+# Install
+
+To install foreman-maintain on foreman or katello instances, please run:
+```
+# yum install rubygem-foreman_maintain
+```
+
+Foreman-maintain is installed as a dependency in Katello 3.7 and up.
+
+# Service management
```

Comment:
This should probably be a `##`

Reply:
I kept these as single `#`s since otherwise the side menu collapses under the last `#` and makes navigation harder (all of these will be collapses on the left under `Foreman Maintain`

Reply:
That's sad :/ and not that logical for a document but OK